.travis.yml

```diff
@@ -225,6 +225,46 @@ jobs:
       sudo docker cp -a ${TRAVIS_BUILD_DIR} mobydick:/tmp                                                               \
       &&                                                                                                                \
       sudo docker start -a mobydick
+  - stage: interface64
```

Comment:
same as the other PR, lot of code duplication here :/

Reply:
No sure to get what you mean: AFAIK, in yml, no way to "factorize" code or "make functions", or, I just don't know how to do that. You need to add job stage by stage as yml grammar impose it, or, you could put all jobs in one stage but then all config are mixed (difficult to debug when pb shows up) and huge-logs blow up (can't access traces when pb show up)

Reply:
most of the operations should be moved into a script
link "setup-env.sh" and called with a bash -v